frontend/internal-packages/agent/src/db-agent/nodes/invokeSchemaDesignToolNode.ts

```diff
@@ -104,8 +104,14 @@ export const invokeSchemaDesignToolNode = async (
     if (schemaResult.isOk()) {
       // Generate DDL statements from the updated schema
       const ddlResult = postgresqlSchemaDeparser(schemaResult.value.schema)
+
+      // Check for DDL generation errors without logging
+      // Errors are handled by returning undefined ddlStatements
+
       const ddlStatements =
-        ddlResult.errors.length > 0 ? undefined : ddlResult.value
+        ddlResult.errors && ddlResult.errors.length > 0
```

Comment:
Does `ddlResult.errors` ever return a falsy value? Was the following not good enough?

```suggestion
        ddlResult.errors.length > 0
```

Reply:
Thank you. It turned out to be unnecessary, so I reverted it on my end.
[refactor: Address PR review comments for dml_execution_logs field](https://github.com/liam-hq/liam/pull/2901/commits/4e6a8066aaee46340dbe310e671fad34e877c7a6)